README.md

```diff
@@ -20,6 +20,13 @@ Somewhere in your `spec_helper` put:
 GovukTest.configure
 ```
 
+If using `Webmock` also add:
+```rb
+WebMock.disable_net_connect!({
```

Comment:
Could we put this in `GovukTest.configure` so it happens automatically if people are using Webmock? Perhaps this Gem should be more opinionated about configuring Webmock since this Gem is about configuring our testing environment for apps.

Reply:
@thomasleese Not all apps making http requests will need the exception for chromedriver (e.g. CCS), though I guess no harm done to have it in unnecessarily.

Do some apps also need to allow localhost?

@edwardkerry This can be more succinctly written as
```
WebMock.disable_net_connect!(allow: "chromedriver.storage.googleapis.com")
```

Reply:
We can do this, and will need to also allow localhost like so:
```
WebMock.disable_net_connect!({
          allow_localhost: true,
          allow: 'chromedriver.storage.googleapis.com'
        })
```
This will overwrite any settings set within the consuming app, but I don't think any active apps have any other exceptions. 

https://github.com/search?p=1&q=org%3Aalphagov+WebMock.disable_net_connect%21%28&type=Code

Reply:
I do like the idea of having this Gem configure Webmock. I think we can include it as part of `configure` for now and split it out later if it becomes problematic with apps that don't need this default configuration.